src/app/Scenes/MyCollection/Screens/Insights/MyCollectionInsights.tests.tsx

```diff
@@ -0,0 +1,108 @@
+import { MyCollectionInsightsTestsQuery } from "__generated__/MyCollectionInsightsTestsQuery.graphql"
+import { StickyTabPage } from "app/Components/StickyTabPage/StickyTabPage"
+import { __globalStoreTestUtils__ } from "app/store/GlobalStore"
+import { flushPromiseQueue } from "app/tests/flushPromiseQueue"
+import { renderWithHookWrappersTL } from "app/tests/renderWithWrappers"
+import React from "react"
+import { useLazyLoadQuery } from "react-relay"
+import { act } from "react-test-renderer"
+import { graphql } from "relay-runtime"
+import { createMockEnvironment } from "relay-test-utils"
+import { AuctionResultsBasedOnArtistsYouCollect } from "./AuctionResultsBasedOnArtistsYouCollect"
+
+jest.unmock("react-relay")
+
+describe("MyCollectionInsights", () => {
+  let mockEnvironment: ReturnType<typeof createMockEnvironment>
+
+  beforeEach(() => {
+    mockEnvironment = createMockEnvironment()
+    __globalStoreTestUtils__?.injectFeatureFlags({ ARShowMyCollectionInsights: true })
+  })
+  const TestRenderer: React.FC = () => {
+    const queryData = useLazyLoadQuery<MyCollectionInsightsTestsQuery>(
+      graphql`
+        query MyCollectionInsightsTestsQuery @raw_response_type {
+          me {
+            ...AuctionResultsBasedOnArtistsYouCollect_me
+          }
+        }
+      `,
+      {}
+    )
+    return (
+      <StickyTabPage
+        tabs={[
+          {
+            title: "test",
+            content: <AuctionResultsBasedOnArtistsYouCollect auctionResults={queryData.me!} />,
```

Comment:
Would this work as well?

```suggestion
            content: <AuctionResultsBasedOnArtistsYouCollect auctionResults={queryData.me} />,
```